### PR TITLE
Add basic commands to title screen

### PR DESCRIPTION
**Pull request by tyler-suard-parker**
Basic commands are needed to make vim more user-friendly.

### CODE REVIEW

File: src/version.c [hunk 1]
```diff
@@ -1424,6 +1424,10 @@ intro_message(
 	N_("type  :help iccf<Enter>       for information "),
 	"",
 	N_("type  :q<Enter>               to exit         "),
+	N_("type  :w<Enter>               to save         "),
+	N_("type  i                       for Insert mode "),
+	N_("type  <Esc>               	  to exit a mode  "),
```
Comment (zeertzjq) [inline src/version.c]:
```suggestion
	N_("type  <Esc>                   to exit a mode  "),
```

**Comment by tyler-suard-parker** [comment]
Related: Issue 14058

**Comment by tyler-suard-parker** [comment]
@zeertzjq Thank you, I could not have done that without you :)

**Comment by chrisbra** [comment]
closing for the reasons mentioned in the related issue: #14058

**Pull request closed without merging**